content/en/monitors/guide/suppress-alert-with-downtimes.md

```diff
@@ -20,9 +20,13 @@ Use the [Downtime API][1] to manage advanced maintenance schedules format, or to
 
 This guide describes how to configure downtimes for the following use cases:
 
-* [Downtime over the weekend](#downtime-over-the-weekend)
-* [Downtime outside of business hours](#downtime-outside-of-business-hours)
-* [Recurring downtime on nth weekday of the month](#recurring-downtime-on-the-nth-weekday-of-the-month)
```

Comment:
Just confirming that it was intentional to add new content here (versus only adjusting the double-spacing).

Reply:
Good call, I'm reverting the change to this file